src/clr-angular/data/datagrid/_datagrid.clarity.scss

```diff
@@ -915,9 +915,146 @@
   }
 
   /*
-        The following classes are used by the renderer when performing internal operations.
-        Using the browser calculations improves rendering performance.
-    */
+   * The following are styles for Detail Pane
+   */
+  .datagrid-outer-wrapper {
+    display: flex;
+    width: 100%;
+    overflow: hidden;
```

Comment:
Be careful here. It looks like vertical scrolling is broken. Seems related to the re-introduction of wrappers of wrappers 😟. Setting a fixed height directly on the datagrid (smaller than the calculated height for pagination) doesn’t allow me to scroll and the footer is hidden.